web/yo/app/views/reportgenerator.html

```diff
@@ -128,5 +128,53 @@ <h3>Review History</h3>
             </tbody>
         </table>
     </div>
+    <hr/>
+    <h3>Reviewed Data</h3>
+    <div class="row" style="margin-top:30px; margin-bottom:30px;">
+        <div class="col-sm-2">Evidence Type:</div>
+        <div class="col-sm-4">
+            <select chosen 
+                    data-placeholder="Enter A type"
+                    ng-model="evidenceType"
+                    ng-options="m.value as m.label for m in evidenceTypes"
+                    class="chosen-select"
+                    ng-change="updateReviewData()">
+                <option value=""></option>
+            </select>
+        </div>
+        <div class="col-sm-2">
+            <button type="button" class="btn btn-default" ng-click="generateEvidences()" ng-bind-html="getReviewButtonContent()"></button>
+        </div>
+        <div class="col-sm-1" ng-if="displayReviewedData">
+            <button type="button" class="btn btn-default" ng-click="downloadReviewedData()"> Download </button>
+        </div>
+    </div>
+    <div ng-if="displayReviewedData" >
+        <table datatable="ng" class="row-border hover" dt-options="reviewedDT.dtOptions">
+            <thead>
+            <tr>
+                <th ng-repeat="x in reviewedDataHeaders[evidenceType]">{{ x }}</th>
+            </tr>
+            </thead>
+            <tbody>
+                <tr ng-repeat="x in reviewedData[evidenceType]">
+                    <td><strong><a href="#/gene/{{x.gene}}">{{ x.gene }}</a></strong></td>
```

Comment:
If you already ng-repeat the header, shouldn't we consider ng-repeat the cell as well?